pkg/controller/controller_test.go

```diff
@@ -1372,6 +1401,9 @@ func runProvisionTest(t *testing.T, k string, tc provisioningTestcase, requested
 		out.Volume.CapacityBytes = int64(80)
 		controllerServer.EXPECT().CreateVolume(gomock.Any(), gomock.Any()).Return(out, nil).Times(1)
 		controllerServer.EXPECT().DeleteVolume(gomock.Any(), gomock.Any()).Return(&csi.DeleteVolumeResponse{}, nil).Times(1)
+	} else if tc.volWithZeroCap {
```

Comment:
I think it would be good to refactor this test to have all of this test case specific behavior passed in as a func() with the rest of the test inputs.  But that can be done as a separate PR.

Reply:
Ok，I think we can add an issue to discuss more detail about refactoring test and resolve it in another PR